op-acceptance-tests/tests/interop/loadtest/l2.go

```diff
@@ -46,8 +63,11 @@ type L2 struct {
 
 func (l2 *L2) DeployEventLogger(ctx context.Context, t devtest.T) {
 	tx, err := l2.Include(ctx, t, txplan.WithData(common.FromHex(bindings.EventloggerBin)))
-	t.Require().NoError(err)
-	l2.EventLogger = tx.Receipt.ContractAddress
+	requireNoErrorUnlessCancelled(t, err)
+	// If the tx is nil, the context was cancelled, and we don't want to NPE during cleanup.
+	if tx != nil {
+		l2.EventLogger = tx.Receipt.ContractAddress
+	}
```

Comment:
My gut feeling is that the explicit `require.NoError` might be better to identify timing problems without risking stranger errors later, but feel free to override that.

Reply:
I was encountering test failures with a short timeout (~10s) but suppose that might be contrived. Will revert this specific diff for now.